css/styles.css

```diff
@@ -71,6 +71,109 @@ header {
 }
 /* END header by Daniel */
 
+/* BEGIN section-3 by Tatsiana */
+.sc-3 {
+  display: flex;
+  -webkit-box-pack: center;
+  justify-content: center;
+  width: 100%;
+  padding: 0px;
+  position: relative;
+  box-shadow: none;
+  background: none;
+  z-index: 1;
+  flex: 0 0 auto;
+}
+.container3 {
+  display: flex;
+  flex-direction: column;
+  -webkit-box-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  align-items: center;
+  width: 100%;
+  max-width: 1280px;
+  padding: 10px 0px;
+  margin: 0px auto;
+}
+.content {
+  display: flex;
+  flex-direction: column;
+  -webkit-box-align: center;
+  align-items: center;
+  box-shadow: rgb(0 0 0 / 8%) 0px 0px 53px;
+  padding: 50px;
+  margin-bottom: 50px;
+}
+.content-1 {
+  display: flex;
+  flex-direction: column;
+  -webkit-box-align: center;
+  align-items: center;
+  width: 100%;
+  margin: 0px 0px 50px;
+}
+.title {
+  font-size: 24px;
+  line-height: 140%;
+  margin-bottom: 20px;
+  text-transform: uppercase;
+  font-family: "Work Sans", sans-serif;
+  font-weight: 700;
+  color: rgb(28, 34, 148);
+  text-align: left;
+  max-width: none;
+}
+.text {
```

Comment:
@TatsianaRud I was reviewing the code and this part with .text class had been used by senait before so it didn't work on my browser. in other words the part took senait .text attributes and ran it to your part of the code.
It would be better if you use another class name more related to the section you working on. for example class ="sc3-text"

Reply:
nice catch Moamin